code/modules/research/designs/limbgrower_designs.dm

```diff
@@ -197,6 +197,39 @@
 	build_path = /obj/item/organ/internal/lungs/ethereal
 	category = list(SPECIES_ETHEREAL)
 
+//Hemophage organs. Bubber Edit.
+/datum/design/hemophage_heart
+	name = "Pulsating Tumor"
+	id = "hemophageheart"
+	build_type = LIMBGROWER
+	reagents_list = list(/datum/reagent/medicine/c2/synthflesh = 10, /datum/reagent/blood = 10)
+	build_path = /obj/item/organ/internal/heart/hemophage
+	category = list(SPECIES_HEMOPHAGE)
+
+/datum/design/hemophage_liver
+	name = "Corrupted Liver"
+	id = "hemophageliver"
+	build_type = LIMBGROWER
+	reagents_list = list(/datum/reagent/medicine/c2/synthflesh = 10, /datum/reagent/blood = 10)
+	build_path = /obj/item/organ/internal/liver/hemophage
+	category = list(SPECIES_HEMOPHAGE)
+
+/datum/design/hemophage_stomach
+	name = "Corrupted Stomach"
+	id = "hemophagestomach"
+	build_type = LIMBGROWER
+	reagents_list = list(/datum/reagent/medicine/c2/synthflesh = 10, /datum/reagent/blood = 10)
+	build_path = /obj/item/organ/internal/stomach/hemophage
+	category = list(SPECIES_HEMOPHAGE)
+
+/datum/design/hemophage_tongue
+	name = "Corrupted Tongue"
+	id = "hemophagetongue"
+	build_type = LIMBGROWER
+	reagents_list = list(/datum/reagent/medicine/c2/synthflesh = 10, /datum/reagent/blood = 10)
+	build_path = /obj/item/organ/internal/tongue/hemophage
+	category = list(SPECIES_HEMOPHAGE)
+
```

Comment:
Add a bubber edit end here, because when I go to review conflicts later I may not see the first part when skimming, and it helps to know what was not from us anyways.

Reply:
Ignore this comment if you do break these additions into a new file.